We do not need to check source code quality for this project.

--- deprecation-2.1.0/test-requirements.txt.orig
+++ deprecation-2.1.0/test-requirements.txt
@@ -1,3 +1,2 @@
-flake8>=3.2
 packaging>=16.8
 unittest2>=1.1;python_version<'3.5'
